src/common/utils/model.ts

```diff
@@ -50,16 +48,12 @@ export function fetchAllStateProjections(snapshotUrl: string | null = null) {
     const all = await new Api(snapshotUrl).fetchAggregatedSummaryWithTimeseries(
       RegionAggregateDescriptor.STATES,
     );
-    return all
-      .filter(
-        summaryWithTimeseries => stateCode(summaryWithTimeseries) !== undefined,
-      )
-      .map(summaryWithTimeseries => {
-        return new Projections(
-          summaryWithTimeseries,
-          stateCode(summaryWithTimeseries)!,
-        );
-      });
+    return all.map(summaryWithTimeseries => {
+      return new Projections(
```

Comment:
Hrm...  I think the filter you removed is what allowed us to add territories in the backend without breaking the frontend.

Reply:
yeah that makes sense. I think I need to check if the state code is in the states object instead (because now you'll never have a null state code, but could run into problems elsewhere)